lib/managers/folders.js

```diff
@@ -301,6 +301,21 @@ Folders.prototype.deleteMetadata = function(folderID, scope, template, callback)
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+/**
+ * Permanently deletes an folder that is in the trash. The item will no longer exist in Box. This action cannot be undone
+ *
+ * API Endpoint: '/folders/:folderID/trash/'
```

Comment:
Super minor nitpick, but would you mind removing the trailing slash after `/trash` here?